docs/topics/proc_configuring-jul-logging-vertx.adoc

```diff
@@ -0,0 +1,19 @@
+
+[#_configure_jul_logging_vertx]
+= Configuring logging for your {VertX} application using JUL
+To configure debug logging for your {VertX} application using JUL:
+
+// is it just `properties` file , more accurate designation
+* specify the `java.util.logging.config.file` system property in the properties file of your project.
```

Comment:
Capitalize first word since you're using full sentences.

Reply:
What's this "properties file of your project"?

Reply:
@rhoads-zach Thanks, Fixed.
@Ladicek AFAIK, that would be the `application.properties` file. But, I agree, that wording is super vague:)